src/distrs/pois.jl

```diff
@@ -19,6 +19,16 @@ poislogpdf(λ::T, x::T) where {T <: Real} = xlogy(x, λ) - λ - loggamma(x + 1)
 
 poislogpdf(λ::Number, x::Number) = poislogpdf(promote(float(λ), x)...)
 
+function poislogcdf(λ::T, x::T) where {T<:Real}
+    return loggamma(floor(x + 1), float(λ)) - logfactorial(floor(x))
```

Comment:
Very minor detail, but it seems a little odd to me use `logfactorial` here together with `loggamma`. In particular when we also use `loggamma(x+1)` in the pdf.